integration_tests/src/main/python/cast_test.py

```diff
@@ -172,68 +172,18 @@ def test_cast_string_date_non_ansi():
 
 
 @pytest.mark.parametrize('data_gen', [StringGen(date_start_1_1_1),
-                                      StringGen(date_start_1_1_1 + '[ |T][0-3][0-9]:[0-6][0-9]:[0-6][0-9]'),
-                                      StringGen(date_start_1_1_1 + '[ |T][0-3][0-9]:[0-6][0-9]:[0-6][0-9]\.[0-9]{0,6}Z?')
+                                      StringGen(date_start_1_1_1 + '[ T][0-3][0-9]:[0-6][0-9]:[0-6][0-9]'),
```

Comment:
[ |T] => [ T]
"[ |T]" represents threee optional chars: ' ', '|' and 'T'.
Here "[ T]" is intended.